cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1019,6 +1022,42 @@ def make_resource(self, data, **kwargs):
         return ClusterDevSettings(**data)
 
 
+# ---------------------- Health Checks ---------------------- #
+
+
+class GpuHealthCheckSchema(BaseSchema):
+    """Represent the schema of gpu health check."""
+
+    enabled = fields.Bool(metadata={"update_policy": UpdatePolicy.SUPPORTED})
+
+    @post_load
+    def make_resource(self, data, **kwargs):
+        """Generate resource."""
+        return GpuHealthCheck(**data)
+
+
+class SlurmQueueHealthChecksSchema(BaseSchema):
```

Comment:
Is it needed to have two different schemas, for queue and for compute resource? I was expecting to be the same

Reply:
Ok. I was thinking about the future expansions (e.g. queue dedicated health checks) but I agree with you: it's out of the scope for this feature at the moment.